### DISCUSSION TITLE
sdist is missing tests

### DISCUSSION THREAD

**Discussion by mtelka** [encode/httpx#2926] [Potential Issue] [answered]
I'm packaging `httpx` for OpenIndiana.  By default we create packages from sdist (downloaded from PyPI) and we run tests to make sure the package works properly.  Unfortunately, the sdist package at PyPI is missing tests.  Could you please consider to add tests to sdist?

Thank you.

**Comment by karpetrosyan** [comment, marked as answer]
Feel free to open a pull request! It's going to be very similar, or exact, like https://github.com/encode/httpcore/pull/799.

**Comment by mtelka** [comment]
Done: https://github.com/encode/httpx/pull/2927
